### PR TITLE
[CWS] Do not generate anomaly during workload startup

### PR DESCRIPTION
**Pull request by lebauce**
### Reviewer's Checklist

- [ ] If known, an appropriate milestone has been selected; otherwise the `Triage` milestone is set.
- [ ] Use the `major_change` label if your change either has a major impact on the code base, is impacting multiple teams or is changing important well-established internals of the Agent. This label will be use during QA to make sure each team pay extra attention to the changed behavior. For any customer facing change use a releasenote.
- [ ] A [release note](https://github.com/DataDog/datadog-agent/blob/main/docs/dev/contributing.md#reno) has been added or the `changelog/no-changelog` label has been applied.
- [ ] Changed code has automated tests for its functionality.
- [ ] Adequate QA/testing plan information is provided if the `qa/skip-qa` label is not applied.
- [ ] At least one `team/..` label has been applied, indicating the team(s) that should QA this change.
- [ ] If applicable, docs team has been notified or [an issue has been opened on the documentation repo](https://github.com/DataDog/documentation/issues/new).
- [ ] If applicable, the `need-change/operator` and `need-change/helm` labels have been applied.
- [ ] If applicable, the `k8s/<min-version>` label, indicating the lowest Kubernetes version compatible with this feature. 
- [ ] If applicable, the [config template](https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml) has been updated.

### CODE REVIEW

**Review by brycekahle — APPROVED** [review]

File: pkg/security/security_profile/profile/manager.go [hunk 1]
```diff
@@ -588,45 +596,41 @@ func (m *SecurityProfileManager) LookupEventInProfiles(event *model.Event) {
...
 
 	if newEntry {
 		profile.lastAnomalyNano = event.TimestampRaw
-		m.eventFiltering[event.GetEventType()][NotInProfile].Inc()
-	} else {
-		m.eventFiltering[event.GetEventType()][InProfile].Inc()
 	}
 
-	return true, nil
+	return true, newEntry, nil
```
Comment (Gui774ume) [inline pkg/security/security_profile/profile/manager.go:635]: Shouldn't it be `!newEntry` ?

File: pkg/security/security_profile/profile/manager.go [hunk 2]
```diff
@@ -588,45 +596,41 @@ func (m *SecurityProfileManager) LookupEventInProfiles(event *model.Event) {
 }
 
 // tryAutolearn tries to autolearn the input event. Returns true if the event was autolearned.
```
Comment (Gui774ume) [inline pkg/security/security_profile/profile/manager.go:598]: Should we add a tiny comment to explain the 3 output variables ? (at least the first 2)

**Review by Gui774ume — COMMENTED** [review]
Happy to chat offline about the different points if need be, but overal lgtm !

**Review by Gui774ume — APPROVED** [review]

**Review by safchain — APPROVED** [review]

File: pkg/config/system_probe.go [hunk 3]
```diff
@@ -309,6 +309,7 @@ func InitSystemProbeConfig(cfg Config) {
 	// CWS - Anomaly detection
 	cfg.BindEnvAndSetDefault("runtime_security_config.security_profile.anomaly_detection.event_types", []string{"exec", "dns"})
 	cfg.BindEnvAndSetDefault("runtime_security_config.security_profile.anomaly_detection.minimum_stable_period", 3600)
+	cfg.BindEnvAndSetDefault("runtime_security_config.security_profile.anomaly_detection.workload_warmup_period", 60)
```
Comment (paulcacheux) [inline pkg/config/system_probe.go:312]: We believe this will break the new tests added by @spikat . Maybe set this to 0 in the tests ?

**Review by spikat — APPROVED** [review]

**Pull request merged**